src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java

```diff
@@ -14,7 +15,7 @@
 
     public static void main(String[] args){
 
-        IStateProvider simulation = new JunctionTestProvider();
+        IStateProvider simulation = new RoadNetworkProvider();
```

Comment:
The `JunctionLocationTestStarter` is meant to start the program with `JunctionTestProvider`. Could you roll this back?